### ISSUE TITLE
How do I build the project?

### ISSUE THREAD

**Issue by swoutxd** [BillytheButcher/GABB#14]
I don't know much about visual studio, so I don't really know what to do. How do I compile the .exe file. I did the github replicate thing in visual studio, but I can't just build it, what do I have to do? I didn't really find any solutions on here, most of them are just some kids who haven't ever heard of visual studio.

**Comment by bacontsu** [comment]
my fix is this 
https://github.com/BillytheButcher/GABB/issues/13

but you have to re-do something in the code to bypass the login system, to make it an offline app. its pretty much simple actually, sadly I have no experience in c++.

**Comment by swoutxd** [comment]
Yeah that's the problem I also have no knowledge of c++ so I can't really change the code, I will try to get a friend who is pretty good at c++ to look at it.

**Comment by bacontsu** [comment]
yeah, thats why now im changing into Blacklock's multiboxing program written on c#, hes using low level keyboard and mouse hook but its also his first time trying c#, so I'm kinda fixing the bug as much as i can because I'm still learning c# too.

**Comment by Praaksu** [comment]
There is a Release thingy on the right side of the page. This is the link, here it is compiled for those who need.
https://github.com/BillytheButcher/GABB/releases/tag/0.6.20

**Comment by swoutxd** [comment]
dont you still have to modify it so it doesnt ask for the login and just let u in

**Comment by Praaksu** [comment]
Nah, install the release, you can use it right away :)
